content/learn/secrets-management-in-the-Validated-Patterns-framework.adoc

```diff
@@ -0,0 +1,33 @@
+---
+menu:
+  learn:
+    parent: Values Files
```

Comment:
Should the parent here rather be secrets?

Reply:
Based on Jira [Telcodocs-1597](https://issues.redhat.com/browse/TELCODOCS-1597), I've added the information under the values files section, as it is part of the documentation around values and charts. However, I understand that we're providing detailed information about managing secrets in validating patterns and that there is a separate section dedicated to secrets.
@danmacpherson, could you please confirm if this information should stay under the values section or be moved to the secrets section?

Reply:
I had a think about this and I can see where @mhjacks is coming from. I do think when a customer comes to our site and is looking for information of secrets configuration, they'd be more likely to go to the Secrets section to find this content. Let's put it under Secrets (first sub menu item, above the Hashicorp Vault page) and we'll assess how to better structure our Learn content going forward.

Reply:
Thanks, @danmacpherson @mhjacks, I'll move it under secrets.